client/src/components/Cards/Cards.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import Card from '../Card/Card'
+
+test('loads item', () => {
+  const sampleCards = [
+    { text: 'What is node?', author: 'Stefan Bering' },
+    { text: 'What is React?', author: ' Timur Buchberger' },
+    { text: 'What is styled components?', author: ' Danilo Bräuer' },
+  ]
+
+  render(
+    <main>
+      {sampleCards.map(card => {
+        return (
+          <Card
+            text={card.text.trim()}
+            author={card.author.trim()}
+            key={card.text.trim()}
+          />
+        )
+      })}
+    </main>
+  )
+
+  for (let sample of sampleCards) {
+    const itemText = screen.getByText(sample.text.trim())
+    expect(itemText).toBeInTheDocument()
+    const authorText = screen.getByText(sample.author.trim())
+    expect(authorText).toBeInTheDocument()
+  }
```

Comment:
I don't think you should loop in a test. What you want to test in this component is that it displays all of your Data. Right now you are neither correctly testing your `Cards` component nor your `Card` component. 
We can talk about it in more detail!

Reply:
I think we got in our own way here, the we should test Card for rendering mockData correctly and test Cards for rendering the correct amount of cards (sampleCards.length).